pkg/api/azenvtypes.go

```diff
@@ -310,9 +310,9 @@ var (
 			EtcdDownloadURLBase:              "mcr.microsoft.com/oss/etcd-io/",
 			KubeBinariesSASURLBase:           DefaultKubernetesSpecConfig.KubeBinariesSASURLBase,
```

Comment:
This may also needs to be updated.

Reply:
@jackfrancis @marosset Could you help to confirm whether `KubeBinariesSASURLBase` is not used now? If so, we do not need to update this URL for Mooncake and update our mirror settings to sync the binaries. BTW: This PR will only be merged for AKS. Later I will create a new one to master branch.

https://github.com/Azure/aks-engine/blob/6c1891c93de325ccfc4949c2e696707de3642b23/pkg/engine/params_k8s.go#L89-L101

Reply:
`KubeBinariesSASURLBase` is definitely used if `kubernetesConfig.CustomWindowsPackageURL` is not specified. Looking through the code, it seems like `kubernetesConfig.CustomWindowsPackageURL` is a user-provided configuration, so the normal (default) cluster configuration will exclude it.

So, yes, this line will depend upon `KubeBinariesSASURLBase`:

https://github.com/Azure/aks-engine/blob/master/parts/k8s/kuberneteswindowssetup.ps1#L68

Reply:
Thanks to confirm it, @jackfrancis. We should also update the `KubeBinariesSASURLBase` and make sure our proxy for Mooncake can provide these binaries. @andyzhangx